test/unit/matchers/matchers_test.rb

```diff
@@ -51,104 +50,160 @@ def refute_cmp(expect, actual)
       end
     end
 
-    it "String cmp String" do
-      assert_cmp "happy", "happy"
-      assert_cmp "HAPPY", "happy" # case insensitive
-      refute_cmp "happy", "unhappy"
-      refute_cmp "happy", nil
-    end
+    # Explicitly mark which tests are meant to be skipped
+    describe "test execution" do
+      # Enable verbose test names in output
+      def self.test_order
+        :alpha
+      end
 
-    it "String cmp String w/o ==" do
-      # String, String, op!==, normal op call?
-      skip "TODO: how to test w/ other ops?"
-    end
+      before do
+        @known_skips = ["test_0002_String cmp String w/o ==", "test_0016_should test XXX"]
+      end
 
-    it "String cmp String w/ versions " do
-      assert_cmp "1.0", "1.0"
-      refute_cmp "1.0.0", "1.0"
-      refute_cmp "1.0", nil
-    end
+      it "String cmp String" do
+        assert_cmp "happy", "happy"
+        assert_cmp "HAPPY", "happy" # case insensitive
+        refute_cmp "happy", "unhappy"
+        refute_cmp "happy", nil
+      end
 
-    it "Regexp cmp String" do
-      assert_cmp(/abc/, "xxx abc zzz")
-      refute_cmp(/yyy/, "xxx abc zzz")
-      refute_cmp(/yyy/, nil)
-    end
+      it "String cmp String w/o ==" do
+        skip "TODO: how to test w/ other ops?"
```

Comment:
Is what you just did answering this TODO? And therefore this test can be deleted? I'm not sure what the author expected to complete this TODO.

Reply:
Those were in the code when I started